src/pages/login/LoginPage.tsx

```
@@ -9,6 +9,7 @@ import { oAuthGetToken } from "@/apis/auth";
 import { useNavigate } from "react-router-dom";
 import Path from "@/types/paths";
 import Button from "@/components/button/Button";
+import Error from "@/assets/error/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)